tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py{27,33,34,35,36}-html5lib{99999999,999999999},pypy-html5lib99999999
+envlist = py{27,33,34,35,36}-html5lib{99999999,999999999,1.0b9,1.0b10},pypy-html5lib99999999
```

Comment:
These additional two `html5lib` versions need to get things added to the `deps` clause below. Something like this:

```
    html5lib1.0b9: html5lib==1.0b9
    html5lib1.0b10: html5lib==1.0b10
```

Also, I'm not sure if periods are allowed in environment names--so we should look into that.